types/pg/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for pg 7.14
+// Type definitions for pg 8.6
```

Comment:
Is this a full update to pg v8?

Reply:
The new `DatabaseError` symbol was added in 8.6.0.  I'm actually not sure what to do with the version number in this file.

Reply:
Hmm I'm not entirely sure either.
Could you perhaps investigate what other updates would be necessary to get these definitions up to speed with v8?

Reply:
A quick glance at the [changelog](https://github.com/brianc/node-postgres/blob/master/CHANGELOG.md) doesn't show any breaking changes to the types.  I don't have the time to do a more thorough check at the moment.